packages/deploy-utils/src/utils/s3.ts

```diff
@@ -9,6 +9,7 @@ export const pushToS3 = (bucket: string) => {
     );
     console.log("Successfully deployed to S3");
   } catch (e) {
-    throw Error("Deployment to S3 failed", { cause: e });
+    console.error(e);
+    throw Error("Deployment to S3 failed");
```

Comment:
Just curious, what's this change for?

Reply:
I have no idea. Must have been an artifact from when I was fighting for my life
Deleted!